code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -10,6 +10,7 @@
 	clothing_flags = HAS_UNDERWEAR | HAS_UNDERSHIRT | HAS_SOCKS
 	bodyflags = HAS_SKIN_TONE | HAS_BODY_MARKINGS
 	dietflags = DIET_OMNI
+	taste_sensitivity = TASTE_SENSITIVITY_DULL
```

Comment:
Humans are generally considered the base average of species- They should really just use the normal taste sensitivity.